packages/schema-extract/src/extract-schema.ts

```diff
@@ -21,21 +23,27 @@ export function* extractSchema(basePath: string, filesGlob: string) {
   }
 }
 
+export function createLinkerProgram(files: string[]) {
+  program = typescript.createProgram(files, {});
+}
+
 export function getSchema(filePath: string, exportName: string) {
-  const program = typescript.createProgram([filePath], {});
+  if (!program) {
```

Comment:
Instead of having the program as a global state I'd prefer it to be an argument of getSchema, or maybe even better to have a class:
```
class LinkerSomething {
   program = null;
   constructor(files) {
     this.program = createLinkerProgram();
   }
   getSchema(...) {
     // Use this.program
   }
}
```